SchemaDatabase/SGr/Generic/BaseTypes.xsd

```diff
@@ -454,199 +454,199 @@ https://github.com/SmartgridReady/SGrSpecifications  -->
-      <enumeration value="JOULE_SECONDS" />
+      <!-- <enumeration value="HORSEPOWER" /> -->
+      <!-- <enumeration value="HOURS" /> -->
+      <!-- <enumeration value="HUNDREDTHS_SECONDS" /> -->
+      <!-- <enumeration value="IMPERIAL_GALLONS" /> -->
+      <!-- <enumeration value="IMPERIAL_GALLONS_PER_MINUTE" /> -->
+      <!-- <enumeration value="INCHES" /> -->
+      <!-- <enumeration value="INCHES_OF_MERCURY" /> -->
+      <!-- <enumeration value="INCHES_OF_WATER" /> -->
+      <!-- <enumeration value="JOULE_SECONDS" /> -->
       <enumeration value="JOULES" />
-      <enumeration value="JOULES_PER_DEGREE_KELVIN" />
-      <enumeration value="JOULES_PER_KILOGRAM_DEGREE_KELVIN" />
-      <enumeration value="JOULES_PER_KILOGRAM_DRY_AIR" />
-      <enumeration value="KILO_BTUS" />
-      <enumeration value="KILO_BTUS_PER_HOUR" />
+      <!-- <enumeration value="JOULES_PER_DEGREE_KELVIN" /> -->
+      <!-- <enumeration value="JOULES_PER_KILOGRAM_DEGREE_KELVIN" /> -->
+      <!-- <enumeration value="JOULES_PER_KILOGRAM_DRY_AIR" /> -->
+      <!-- <enumeration value="KILO_BTUS" /> -->
+      <!-- <enumeration value="KILO_BTUS_PER_HOUR" /> -->
       <enumeration value="KILOGRAMS" />
-      <enumeration value="KILOGRAMS_PER_CUBIC_METER" />
-      <enumeration value="KILOGRAMS_PER_HOUR" />
-      <enumeration value="KILOGRAMS_PER_MINUTE" />
-      <enumeration value="KILOGRAMS_PER_SECOND" />
-      <enumeration value="KILOHERTZ" />
-      <enumeration value="KILOOHMS" />
-      <enumeration value="KILOJOULES" />
-      <enumeration value="KILOJOULES_PER_DEGREE_KELVIN" />
-      <enumeration value="KILOJOULES_PER_KILOGRAM" />
-      <enumeration value="KILOJOULES_PER_KILOGRAM_DRY_AIR" />
-      <enumeration value="KILOMETERS_PER_HOUR" />
-      <enumeration value="KILOPASCALS" />
-      <enumeration value="KILOVOLT_AMPERES" />
-      <enumeration value="KILOVOLT_AMPERE_HOURS" />
-      <enumeration value="KILOVOLT_AMPERES_REACTIVE" />
-      <enumeration value="KILOVOLT_AMPERES_REACTIVE_HOURS" />
-      <enumeration value="KILOVOLTS" />
+      <!-- <enumeration value="KILOGRAMS_PER_CUBIC_METER" /> -->
+      <!-- <enumeration value="KILOGRAMS_PER_HOUR" /> -->
+      <!-- <enumeration value="KILOGRAMS_PER_MINUTE" /> -->
+      <!-- <enumeration value="KILOGRAMS_PER_SECOND" /> -->
+      <!-- <enumeration value="KILOHERTZ" /> -->
+      <!-- <enumeration value="KILOOHMS" /> -->
+      <!-- <enumeration value="KILOJOULES" /> -->
+      <!-- <enumeration value="KILOJOULES_PER_DEGREE_KELVIN" /> -->
+      <!-- <enumeration value="KILOJOULES_PER_KILOGRAM" /> -->
+      <!-- <enumeration value="KILOJOULES_PER_KILOGRAM_DRY_AIR" /> -->
+      <!-- <enumeration value="KILOMETERS_PER_HOUR" /> -->
+      <!-- <enumeration value="KILOPASCALS" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERES" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERE_HOURS" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERES_REACTIVE" /> -->
```

Comment:
`KILOVOLT_AMPERES_REACTIVE` wird in einigen Funktionsprofilen und Produkten benutzt - auch in veröffentlichten wie `FP_SGr_Metering_ReactivePowerAC_m_1.0.xml`.

Reply:
Wir können KILOVOLT_AMPERES_REACTIVE auskommentieren